cli/tests/pcluster/config/test_section_cluster/test_cluster_from_file_to_cfn/pcluster.config.ini

```diff
@@ -211,4 +212,5 @@ extra_json = { "cluster" : { "cfn_scheduler_slots" : "cores" } }
 additional_cfn_template = https://test
 tags = {"test": "test"}
 custom_chef_cookbook = https://test
-custom_awsbatch_template_url = https://test
\ No newline at end of file
+custom_awsbatch_template_url = https://test
+disable_hyperthreading = false
```

Comment:
Could you add a new test with `disable_hyperhreading = true` or change this one?